pallets/oracle/src/lib.rs

```diff
@@ -110,39 +122,15 @@ pub mod pallet {
 		}
 	}
 
-	/// Current medianized value for the given key
-	#[pallet::storage]
-	pub type Aggregate<T: Config> =
-		StorageMap<_, Blake2_128Concat, OracleKey, UnsignedFixedPoint<T>>;
-
-	#[pallet::storage]
-	pub type RawValues<T: Config> = StorageDoubleMap<
-		_,
-		Blake2_128Concat,
-		OracleKey,
-		Blake2_128Concat,
-		T::AccountId,
-		TimestampedValue<UnsignedFixedPoint<T>, T::Moment>,
-	>;
-
-	#[pallet::storage]
-	/// if a key is present, it means the values have been updated
-	pub type RawValuesUpdated<T: Config> = StorageMap<_, Blake2_128Concat, OracleKey, bool>;
-
-	/// Time until which the aggregate is valid
-	#[pallet::storage]
-	pub type ValidUntil<T: Config> = StorageMap<_, Blake2_128Concat, OracleKey, T::Moment>;
-
 	/// Maximum delay (milliseconds) for a reported value to be used
 	#[pallet::storage]
 	#[pallet::getter(fn max_delay)]
 	pub type MaxDelay<T: Config> = StorageValue<_, T::Moment, ValueQuery>;
 
 	// Oracles allowed to set the exchange rate, maps to the name
```

Comment:
This comment is wrong now. Please update it accordingly.

Reply:
Done